### DISCUSSION TITLE
Does there need to be a network winner between IOTA and Shimmer? A proposal for a mutually beneficial balance between the two networks.

### DISCUSSION THREAD

**Discussion by rotorstar** [iota-community/Shimmer-Governance#6] [Ideas]
With the introduction of Shimmer, the IOTA Foundation has provided a great proposal on how combined can,

(a) create an incentivized network that is a playground for great new business models to be tested
b) engage the community via a DAO.
c) create a token with future value appreciation.

Hans Moog defined it nicely by saying: _IOTA's crazy little cousin_

Yes, that's exactly what it's supposed to be. And this chart from IF illustrates it once again. 

<img width="923" alt="2021-11-23_10-18-07" src="https://user-images.githubusercontent.com/35960900/143135862-fa078a9c-513c-4479-b666-a3cab834cf14.png">

Unfortunately, in the following days it turned out that the picture and the separation of the networks became more and more cloudy. 
With statements like "Shimmer could become more successful than IOTA" some users asked the justified question: why do we still need IOTA (the network and the token) in the future FOR THE COMMUNITY? 

I have thought about how to leverage both networks together and would like to know your opinion.  I have just quickly created this diagram.

<img width="657" alt="2021-11-23_09-05-11" src="https://user-images.githubusercontent.com/35960900/143140195-b2f8f25d-24fa-4293-a22f-a3e080c49566.png">
_(The percentages are just examples and can of course be adjusted.)_

I would like to explain it (I am deliberately leaving out the topic of NFT in this consideration, as it requires a separate discussion):

Initially we will stake MIOTA to get SMR.

In the later step with the start of the Shimmer network it will also be possible and necessary to stack SMR to get further SMR. 8% Inflation.

I would like to see this distribution split and a portion also go back into MIOTA.

**Motivations:**

The shimmernet is a staging net, meaning that apps will be tested here and perhaps driven by the psychology of curiosity, many more tokens will be in circulation and possibly traded very quickly. It may well be that Shimmer will actually be (temporarily) more successful in terms of sales volume. We do not know.

But I would like to be able to keep the motivation high that a lot of work and trust is still flowing into the STABLE network.

I can only do that if I provide money and stabilize the token price there as well.

The idea is that of the SMR staked shares, say 80% is distributed to the Staker. 5% will be distributed to the Shimmer DAO but 15% must also be transacted for a buyback from MIOTA. I.e. MIOTA must be **_bought back_** from the market (via Smart Contract), which are then available to the IOTA DAO. I.e. the tests on the Shimmernet also promote the stable network.

It can be assumed that the more SMR is staked, the more successful the network is. So the more also flows into the development of the STABLE network (IOTA = gold standard). I.e. standardization, support and winning of larger institutions, which set evenly on stability (EBSI, OMG, LinuxFoundation, Alvarium, etc.).

So far I have heard only two notable concerns about this idea and would like to respond to them:

1) 
Q: Isn't there a particular selling pressure here on $SMR if you have to keep selling these tokens in order to buy MIOTAs in the market.

A: 
1. supply and demand will regulate the price. Only when there are more sellers than buyers can there be selling pressure. 

2. the staking volume is unlikely to be very volatile, so it is likely to be very even buybacks. 

3. the percentages are variable and probably the sweet spot will still be found out

2) 
Q: Is a buy back legally allowed?

A: DeFi companies, like Celsius or Vesper practice this with a different model every day. But of course we should have it legally checked for our case. 

I am glad if you like and discuss this proposal.

**Comment by ACC30** [comment]
I like the idea that stackers get 8%, but at the same time deduct 1% in dao.  in fact, 7% remains for the stackers, and 1% simply goes to the treasury of the Tao for testing.  The DAO, in turn, should also have a shimmer stacking function for the period of writing the code and downtime in testing (the surplus should work).  but the question about smr staking, which is not yet clear to me: when does the accrual take place?  hourly, daily, quarterly?  those I write the code of the project - the stacking is in progress, I wrote the code in 43 days (the numbers are just for example).  now i am ready to stop stacking and use my smr in my project.  how and how much will i get smr in 43 days?  Redemption of iota in the dao shimmer wallet is a great idea, but do not forget that you will need to redeem both stable coins and ether and bitcoin to minimize risks in the season of the general fall of the crypt or the season of blue chips.  the purchase of blue nft is also a stabilization instrument and should be implemented.  the mechanism for transferring nft from other networks is also important.  burn a couple of crypto-banks on the Ethereum network and create them on the iota-shimmer network.  maybe I digress a little from the main topic - sorry.  the thought came and I decided to voice it.
